lib/ModelMixins/TableMixin.ts

```diff
@@ -102,7 +102,20 @@ export default function TableMixin<T extends Constructor<Model<TableTraits>>>(
       const value = super.activeStyle;
       if (value !== undefined) {
         return value;
-      } else if (this.styles && this.styles.length > 0) {
+      } else if (
+        this.styles &&
+        this.styles.length > 0 &&
+        isDefined(this.styles[0].id)
```

Comment:
Why is `isDefined(this.styles[0].id)` needed?

Reply:
Just to get rid of TypeError for `this.tableColumns.find(col => col.name === this.styles[i].id)`

Reply:
I can't remember, It must be residual from an earlier version. All of this is pretty rough.